api_tests/tsconfig.json

```diff
@@ -5,11 +5,14 @@
         "allowSyntheticDefaultImports": true,
         "noImplicitAny": true,
         "lib": ["es5", "es2015", "es2017", "es6", "dom"],
-        "noEmit": true,
+        "noEmit": false,
         "noUnusedLocals": true,
         "noUnusedParameters": true,
         "resolveJsonModule": true,
+        "sourceMap": true,
         "esModuleInterop": true,
+        "outDir": "dist",
```

Comment:
Is this needed?

Reply:
Yes unfortunately, because Jest does not understand "TestEnvironments" that are written in TS.